hatch_build.py

```diff
@@ -206,15 +202,6 @@
     # 0.115.10 fastapi was a bad release that broke our API's and static checks.
     # Related fastapi issue here: https://github.com/fastapi/fastapi/discussions/13431
     "fastapi[standard]>=0.112.4,!=0.115.10",
-    "flask-caching>=2.0.0",
-    # Flask-Session 0.6 add new arguments into the SqlAlchemySessionInterface constructor as well as
-    # all parameters now are mandatory which make AirflowDatabaseSessionInterface incompatible with this version.
-    "flask-session>=0.4.0,<0.6",
-    "flask-wtf>=1.1.0",
```

Comment:
Flask-wtf I'm not 100% sure we can remove from core yet due to possible use in providers manager. Lets see.

Reply:
I don't think we can.

Reply:
We could. Provider's manager currently handles missing flask imports gently - the only thing needed is to improve the `try/except` pattern there to handle it also for wtforms import.

I added a fix for that https://github.com/apache/airflow/pull/48094

Reply:
Ace. Rebased this now, lets see if it's good now